powersimdata/data_access/scenario_list.py

```diff
@@ -148,34 +138,31 @@ def err_message(text):
                 .to_dict("records", into=OrderedDict)[0]
             )
 
+    @verify_hash
     def add_entry(self, scenario_info):
         """Adds scenario to the scenario list file on server.
 
         :param collections.OrderedDict scenario_info: entry to add to scenario list.
+        :return: (*pandas.DataFrame*) -- the updated data frame
         """
-        print("--> Adding entry in %s on server" % self._SCENARIO_LIST)
-        entry = ",".join(scenario_info.values())
-        options = "-F, -v INPLACE_SUFFIX=.bak -i inplace"
-        # AWK parses the file line-by-line. When the entry of the first column is
-        # equal to the scenario identification number, the entire line is replaced
-        # by the scenaario information.
-        program = "'{if($1==%s) $0=\"%s\"};1'" % (
-            scenario_info["id"],
-            entry,
-        )
-        command = "awk %s %s %s" % (options, program, self._server_path)
-
-        err_message = "Failed to add entry in %s on server" % self._SCENARIO_LIST
-        _ = self._execute_and_check_err(command, err_message)
-
-    def delete_entry(self, scenario_info):
+        table = self.get_scenario_table()
```

Comment:
This seemed like it should be easier to do in pandas, but it's the only way I got it to work. Let me know if you have suggestions.